### PR TITLE
Feature/scenarios pill

### PR DESCRIPTION
**Pull request by weberjavi**
### Overview

Components needed to display the scenarios list have been created (`progressBar` and `Scenarios:item and Scenarios:list`)

### Designs

image

### Testing instructions

`yarn && yarn storybook:dev`
Check the new components created (`progressBar`, `Scenarios: Item & List` )

### Feature relevant tickets

https://www.pivotaltracker.com/story/show/176244336

### CODE REVIEW

**Review by mbarrenechea — CHANGES_REQUESTED** [review]

File: app/components/scenarios/item/component.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const Template: Story<ItemProps> = ({ ...args }: ItemProps) => (
+  <Item {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  name: 'Scenario 1',
+  progress: 20,
+  scenarioState: 'running',
```
[the remark this answers is not in the document]
Comment (weberjavi) [reply app/components/scenarios/item/component.stories.tsx:19]: ⚡

File: app/components/scenarios/item/component.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      [className]: !!className,
+    })}
+  >
+    <div className="flex flex-col flex-grow pl-8 bg-gray-700 rounded-l-3xl">
+      <div className="flex items-center flex-grow pr-5 bg-gray-700 rounded-l-3xl">
+        <div className="flex items-center flex-grow max-h-full space-x-4 text-lg text-white">
+          <section className="flex-grow">
+            <div className="flex flex-row items-center">
+              <div>
+                {warnings && (
```
[the remark this answers is not in the document]
Comment (weberjavi) [reply app/components/scenarios/item/component.tsx:62]: ⚡

File: app/components/scenarios/list/component.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  items: ItemProps[];
+}
+
+export const List: React.FC<ListProps> = ({
+  className,
+  items = [],
+}: ListProps) => (
+  <div
+    className={cx({
+      'grid grid-rows-3 gap-3': true,
```
[the remark this answers is not in the document]
Comment (weberjavi) [reply app/components/scenarios/list/component.tsx:18]: ⚡

**Comment by weberjavi** [comment]
☝🏻 ready again

**Review by mbarrenechea — APPROVED** [review]
Good job

**Pull request merged**